src/api/data/pamelding.ts

```diff
@@ -1,35 +1,46 @@
-import {z} from 'zod'
+import { z } from 'zod'
 
 export enum Tiltakstype {
-  ARBFORB = 'ARBFORB', // Arbeidsforberedende trening / AFT
-  ARBRRHDAG = 'ARBRRHDAG', // Arbeidsrettet rehabilitering / ARR
-  AVKLARAG = 'AVKLARAG', // Avklaring
-  INDOPPFAG = 'INDOPPFAG', // Oppfølging
-  DIGIOPPARB = 'DIGIOPPARB', // Digitalt oppfølgingstiltak
-  GRUFAGYRKE = 'GRUFAGYRKE', // Fag og yrkesopplæring gruppe / er kurs
-  GRUPPEAMO = 'GRUPPEAMO', // Arbeidsmarkedsopplæring i gruppe / er kurs
-  JOBBK = 'JOBBK', // Jobbklubb / er kurs
-  VASV = 'VASV' // Varig tilrettelagt arbeid (skjermet virksomhet) / VTA
+	ARBFORB = 'ARBFORB', // Arbeidsforberedende trening / AFT
+	ARBRRHDAG = 'ARBRRHDAG', // Arbeidsrettet rehabilitering / ARR
```

Comment:
VI må se på formattering :)

Reply:
Det har alltid vært litt texas i frontend 😅

Reply:
haha, ja Magnus har satt opp intelij for å fungere med lint og prettier på save :)

Reply:
Hvis dere har bestemt dere for regler så er det vel på tide å sette opp en sjekk på det, og formatere de filene som er feil nå. Jeg antok at det ikke var så viktig siden formateringen er forskjellig så mange steder.